src/main/java/com/smartling/marketo/sdk/rest/transport/JaxRsHttpCommandExecutor.java

```diff
@@ -22,12 +23,17 @@
 import java.lang.reflect.Type;
 import java.net.URLEncoder;
 import java.util.HashMap;
+import java.util.List;
 import java.util.Map;
 import java.util.Map.Entry;
+import java.util.Optional;
 
 public class JaxRsHttpCommandExecutor implements HttpCommandExecutor {
     private final static ObjectMapper OBJECT_MAPPER = new ObjectMapper();
 
+    private static final String RATE_LIMIT_CODE = "606";
+    private static final String CONCURRENT_REQUEST_LIMIT_CODE = "615";
+
```

Comment:
There is also `607 - Daily quota reached`
It might be reached if client uses the same creds for another integration with huge amount of requests per day
I've seen this error only once, but it will be nice to have a separate exception for this error to allow bgtasks to consider it as a recoverable exception and not fail submission.
May be it should be done in a separate task